portal-web/test/functional/com/liferay/portalweb/macros/SignIn.macro

```diff
@@ -39,6 +39,26 @@ definition {
 		AssertElementNotPresent(locator1 = "Button#I_AGREE");
 	}
 
+	macro portalInstanceSetUp {
+		var portalURL = PropsUtil.get("portal.url");
+		var portalURL = StringUtil.replace("${portalURL}", "http://", "");
+		var portalURL = StringUtil.replace("${portalURL}", ":8080", "");
+
+		var testName = selenium.getTestName();
```

Comment:
@vicnate5  I had to do this because its not supported to call a *.function with a return type currently, its allowed if you call the method in BaseWebDriverImpl directly